extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -105,6 +105,9 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
 
     /** @private {../../../src/service/xhr-impl.FetchResponseHeaders} */
     this.responseHeaders_ = null;
+
+    /** @private {?{width, height}} */
```

Comment:
Given this falls back to result of this.getIntersectionLayoutBox() we should make the type match {!./layout-rect.LayoutRectDef}